common/races.cpp

```diff
@@ -1420,26 +1420,26 @@ uint32 GetPlayerRaceBit(uint16 race_id) {
 
 uint16 GetRaceIDFromPlayerRaceValue(uint32 player_race_value) {
 	switch (player_race_value) {
-		case PLAYER_RACE_HUMAN:
-		case PLAYER_RACE_BARBARIAN:
-		case PLAYER_RACE_ERUDITE:
-		case PLAYER_RACE_WOOD_ELF:
-		case PLAYER_RACE_HIGH_ELF:
-		case PLAYER_RACE_DARK_ELF:
-		case PLAYER_RACE_HALF_ELF:
-		case PLAYER_RACE_DWARF:
-		case PLAYER_RACE_TROLL:
-		case PLAYER_RACE_OGRE:
-		case PLAYER_RACE_HALFLING:
-		case PLAYER_RACE_GNOME:
+		case Race::Human:
+		case Race::Barbarian:
+		case Race::Erudite:
+		case Race::WoodElf:
+		case Race::HighElf:
+		case Race::DarkElf:
+		case Race::HalfElf:
+		case Race::Dwarf:
+		case Race::Troll:
+		case Race::Ogre:
+		case Race::Halfling:
+		case Race::Gnome:
 			return player_race_value;
-		case PLAYER_RACE_IKSAR:
+		case Race::Aviak:
```

Comment:
These are wrong starting here, the `player_race_value` doesn't correspond to race id for this function

Reply:
Good call - I need the script to ignore some of the other constant values